{{cookiecutter.project_slug}}/.github/pull_request_template.md

```diff
@@ -15,9 +15,7 @@
 
 ## Checklist:
 - [ ] Add test cases to all the changes you introduce
-- [ ] Run `inv lint` locally to ensure all linter checks pass
-- [ ] Run `inv test` locally to ensure all test cases pass
-- [ ] Run `inv secure` locally to ensure no major vulnerability is introduced
+- [ ] Run `poetry run pytest` locally to ensure all linter checks pass
```

Comment:
Actually, we still use the old checklist. By running `inv test`, it'll call `poetry run pytest`. Thus, I might need your help to change it back.